src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -720,4 +748,11 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges {
       }
     });
   }
+
+  /**
+   * Adds the current row information to the querystring of the table URL
+   */
+  private _updateFilteredTableUrl(): void {
+    this.filteredTableUrl = this._appStateService.rowId ? `${this.formMetadata.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${this._appStateService.rowId}` : this.filteredTableUrl;
```

Comment:
I know this was just moved from another file, but I don't like that it's self-referencial. If you have a record selected, then deselect that record, this URL will still point to it. I think instead of setting back to `this.filteredTableUrl` you should just make it go to the base table (or, alternatively, not include this option?)

Reply:
@cinchy-m Do you prefer to disable the button if no record is selected? Or link to the base table?

Reply:
Yeh I don't want to dynamically add/remove items from a menu. It needs to just be disabled if no record is selected.

I posted a question in the chat about the table link usage as I'm contemplating moving that link to this menu so you'd have both but in one place - "View Table" and "View record in table". We can leave that for now though.